leeger/league_loader/YahooLeagueLoader.py

```diff
@@ -125,7 +128,8 @@ def __getMatchupType(self, yahooMatchup: YahooMatchup) -> MatchupType:
                 if team1Id in self.__yearToTeamIdHasLostInPlayoffs[yahooMatchup.league.season] \
                         and not self.__yearToTeamIdHasLostInPlayoffs[yahooMatchup.league.season][team1Id] \
                         and team2Id in self.__yearToTeamIdHasLostInPlayoffs[yahooMatchup.league.season] \
-                        and not self.__yearToTeamIdHasLostInPlayoffs[yahooMatchup.league.season][team2Id]:
+                        and not self.__yearToTeamIdHasLostInPlayoffs[yahooMatchup.league.season][team2Id] \
```

Comment:
this is the only change for this one, other changes are from issue #28.
Also, though wasn't sure if the matchups that are consolation should fall into the code below, which is what would happen as it is now

Reply:
this is_consolation field is likely the magic fix here... I actually had multiple people reach out to me from Reddit about this happening so it is a known issue.

Reply:
yea it works, just now though all the matchups that are consolation will count as a PLAYOFF matchup I'm assuming?  I'm not sure what happens to these further down the line if this will corrupt the actual playoff stats if consolation is grouped into them or if that is the correct way to handle it, if they are even used or not I'm not sure either. :)  
But it stops the multiple championship validation error from happening with just this one change.

Reply:
Consolation matchups should count as playoff matchups.

Any matchup that determines final standing _should_ count as a playoff matchup in my opinion.

Perhaps down the line there can be another type of matchup (MatchupType.CONSOLATION) if this becomes an issue and people want those segregated but for now I believe just counting them as playoff matchups is sufficient.